lib/pages/child/calendar_page.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:date_utils/date_utils.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:table_calendar/table_calendar.dart';
+
+import 'package:fokus/model/db/date/date.dart';
+import 'package:fokus/model/ui/user/ui_child.dart';
+import 'package:fokus/model/ui/plan/ui_plan.dart';
+
+import 'package:fokus/logic/calendar_cubit.dart';
+import 'package:fokus/services/app_locales.dart';
+import 'package:fokus/utils/theme_config.dart';
+import 'package:fokus/utils/calendar_utils.dart';
+import 'package:fokus/widgets/chips/attribute_chip.dart';
+import 'package:fokus/widgets/cards/item_card.dart';
+import 'package:fokus/widgets/segment.dart';
+
+class ChildCalendarPage extends StatefulWidget {
+	@override
+	_ChildCalendarPageState createState() => new _ChildCalendarPageState();
+}
+
+class _ChildCalendarPageState extends State<ChildCalendarPage> with TickerProviderStateMixin {
+	static const String _pageKey = 'page.childSection.calendar';
+	AnimationController _animationController;
+	CalendarController _calendarController;
+
+	final Duration _animationDuration = Duration(milliseconds: 250);
+
+	@override
+	void initState() {
+		super.initState();
+		_calendarController = CalendarController();
+		_animationController = AnimationController(vsync: this, duration: _animationDuration);
+		_animationController.forward();
+	}
+
+	@override
+	void dispose() {
+		_calendarController.dispose();
+		_animationController.dispose();
+		super.dispose();
+	}
+
+	@override
+	Widget build(BuildContext context) {
+		return Scaffold(
+			appBar: AppBar(
+				title: Text(AppLocales.of(context).translate('$_pageKey.header.title')),
+			),
+			body: Column(
+				crossAxisAlignment: CrossAxisAlignment.start,
+				children: [
+					AppSegments(
+						segments: [
+							Padding(
+								padding: EdgeInsets.symmetric(horizontal: AppBoxProperties.screenEdgePadding),
+								child: BlocBuilder<CalendarCubit, CalendarState>(
+									builder: (context, state) {
+										if (state.children == null)
+											context.bloc<CalendarCubit>().loadInitialData();
+										return _buildCalendar(state.events, state.children);
+									},
+								)
+							),
+							Padding(
+								padding: EdgeInsets.only(top: AppBoxProperties.sectionPadding),
+								child: _buildPlanList()
+							)
+						]
+					)
+				]
+			)
+		);
+	}
+
+	Widget _buildPlanList(){
```

Comment:
You can also extract this method from both pages to util

Reply:
With cubit inside? 🤔 
Also this chunk is different for both calendar pages:
![obraz](https://user-images.githubusercontent.com/36575365/92766657-b37a4480-f396-11ea-8b02-520cee32bf67.png)

So I can get the Segment widget out I guess, but there would be a lot of arguments.. (remember that texts for children can be changed later to better fit them).

Reply:
Okay, lets stick with this than